square/js/action.js

```
@@ -0,0 +1,136 @@
+
+			newDropdown.appendChild( option );
+		}
+
+		intervalCountSetting.parentNode.insertBefore( newDropdown, intervalCountSetting.nextSibling );
+
+		intervalCountSetting.remove();
+		clone.querySelector( '[name*="[post_content][interval]"]' )?.remove();
+		settingWrapper.parentNode.insertBefore( clone, settingWrapper.nextSibling );
+
+		const label = newDropdown.closest( '.frm_trans_sub_opts' )?.querySelector( 'label' );
+		if ( label && label.textContent.includes( 'Repeat Every' ) ) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)